test/e2e/stack.go

```diff
@@ -27,6 +32,11 @@ type e2eCfnStack struct {
 	cfn                    *cloudformation.CloudFormation
 	iam                    *iam.IAM
 	iamRolesAnywhereCACert []byte
+	region                 string
+}
+
+type hybridCfnTemplateVars struct {
+	IRASupported bool
```

Comment:
nit: avoid non standard acronyms when possible. Also, naming things after what they do instead of what they are use for tends to draw better boundaries between components
```suggestion
	IncludeRolesAnywhere bool
```

Reply:
Done